frontend/integration-tests/tests/crud.scenario.ts

```diff
@@ -6,13 +6,16 @@ import * as _ from 'lodash';
 import { execSync } from 'child_process';
 import { OrderedMap } from 'immutable';
 
-import { appHost, testName, checkLogs, checkErrors } from '../protractor.conf';
+import { appHost, testName, checkLogs, checkErrors, SLOW_BACKEND } from '../protractor.conf';
 import * as crudView from '../views/crud.view';
 import * as yamlView from '../views/yaml.view';
 import * as namespaceView from '../views/namespace.view';
 import * as createRoleBindingView from '../views/create-role-binding.view';
 
-const K8S_CREATION_TIMEOUT = 15000;
+const K8S_CREATION_TIMEOUT = 15000 * SLOW_BACKEND;
+const YAML_EDITOR_TIMEOUT = K8S_CREATION_TIMEOUT;
+const K8S_EDIT_TIMEOUT = K8S_CREATION_TIMEOUT * 2;
+const K8S_DELETION_TIMEOUT = K8S_CREATION_TIMEOUT * 6;
```

Comment:
I'm not sure we should start setting separate timeouts for each test. I would rather set a reasonable default and then only specify a timeout for tests that are expected to be slow.

Reply:
Removed all those individual timeout settings. PTAL.